tests/mapping/test_mapping_script.py

```diff
@@ -1,17 +1,20 @@
+import unittest
 import rdflib
 import datetime
 import os
 import yaml
 from pathlib import Path
+import re
 
-from lebedigital.mapping.emodul_mapping import generate_knowledge_graph, get_date_time_value
+from lebedigital.mapping.emodul_mapping import placeholderreplacement
 
 
 def query_and_test_result(typeOfQuery, predicate, prop, g):
     print(f"""Query for {typeOfQuery}:""")
     q = f"""
             SELECT ?o WHERE {{
                     {predicate} {prop} ?o .
+                    FILTER (strstarts(str(), {predicate})) 
```

Comment:
What is that query supposed to do?

Reply:
This query should return the objects which starts with something (like diameter, height, weight, etc) as they are followed with a unique id in the current version and we cannot call them by sending the parameter directly.

Reply:
IMHO, this type of query is not useful, since as soon as we have more than one test (with each having a diameter), this is returning list of diameters without any relation. As such, this type of query should be replaced by searching for a specific test (e.g. using the human readible id) and then return the diameter of that test.